test/functional/query-runner/async-dispose.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from "chai"
+import "reflect-metadata"
+import { DataSource } from "../../../src/data-source/DataSource"
+import {
+    closeTestingConnections,
+    createTestingConnections,
+    reloadTestingDatabases,
+} from "../../utils/test-utils"
+import { EntityManager } from "../../../src"
+
+describe("query runner > async dispose", () => {
+    let dataSources: DataSource[]
+    before(async () => {
+        dataSources = await createTestingConnections({
+            entities: [__dirname + "/entity/*{.js,.ts}"],
+            enabledDrivers: ["postgres"],
+        })
+    })
+    beforeEach(() => reloadTestingDatabases(dataSources))
+    after(() => closeTestingConnections(dataSources))
+
+    it("should release query runner", () =>
```

Comment:
Do you think it's valuable to add few cases?
eg. execute a query and verify the automatic release at the end.